index.js

```diff
@@ -125,6 +125,20 @@ const getCountriesByFamousFor = (famousThing) => {
   );
 };
 
+/** 
+ * Returns an array of objects containing all countries, each containing `country`, `capital`,
+ * `currency`, `native_language`, `famous_for`, `phone_code`, `flag` and `drive_direction` filtered by `is_landlocked`
+ * @param {'true' | 'false'} isLandLocked  Country is said to be landlocked when it is surrounded on all sides by one or more countries
```

Comment:
The way you defined that param would mean that user needs to provide one of two `String` but actually it has to be `Boolean`

Reply:
Thanks ! will make the change.